kbase/collectd/docker_stats.py

```diff
@@ -214,7 +213,16 @@ def read_func():
         stats = get_stats(container)
         meta = build_metadata(container)
         values = [stats[k] for k in TYPE_INSTS]
-        collectd.Values(type='docker', type_instance=container.id, plugin='docker_stats', meta=meta).dispatch(values=values, meta=meta)
+        # Pack the image_name, container.name and container.short_id into the type_instance field
+        # Parse out the name:tag of the running image from the image object
+        # You can get the name but not tag of image from container attrs, so this
+        # seems to be easiest method
+        instance = {"image": IMG_REGX.search(str(container.image)).group(1),
+                    "name": container.name,
+                    "short_id": container.short_id}
+        type_instance = " ".join(["{0}={1}".format(k, v) for k, v in instance.items()])
```

Comment:
so, now the elastic index contains a field that holds multiple fields? Is that what I am understanding?

Reply:
@ajtritt Can you merge this? I've got the parser tested and deployed
https://github.com/kbase/logstash/blob/39841dd11779680eeea3563341b0b9e351ef71ec/pipeline/20collectd_docker#L3